jac-uk/apply
# fix: #1240 Browser info issue on firefox

## What's included?
We collect browser info when a QT is started. This fixes an error on firefox

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
(see @HalcyonJAC visuals in the comments below)
You will need to set up a QT in admin.
On firefox:
- Log in to Apply > Qualifying Tests
- Click to start the QT
- You should be able to start the QT without any issues

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work
---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
